pytket/phir/phirgen.py

```diff
@@ -544,7 +556,7 @@ def get_decls(qbits: set["Qubit"], cbits: set[tkBit]) -> list[dict[str, str | in
     decls += [
         {
             "data": "cvar_define",
-            "data_type": "u32",
+            "data_type": f"i{WORDSIZE}",
```

Comment:
Changing to i64 as the default as per the spec. Tested with previous version of pytket to check that i32 works as well.